nemo/collections/tts/torch/examples.py

```diff
@@ -0,0 +1,50 @@
+import torch
```

Comment:
Can you update https://github.com/NVIDIA/NeMo/tree/main/nemo/collections/tts/torch as well?

Reply:
Done